SharedCryptoPkg/SharedCryptoPkg.dec

```diff
@@ -0,0 +1,51 @@
+## @file
+# Package declaration file for SharedCryptoPkg Library and Driver
+#
+# Copyright (c) 2019, Microsoft Corporation
+
+# All rights reserved.
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+# 1. Redistributions of source code must retain the above copyright notice,
+# this list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+# this list of conditions and the following disclaimer in the documentation
+# and/or other materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+# IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+# INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+# BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+# DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+# LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+# OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+# ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+##
+
+[Defines]
+  DEC_SPECIFICATION              = 0x00010005
+  PACKAGE_NAME                   = SharedCryptoPkg
+  PACKAGE_UNI_FILE               = SharedCryptoPkg.uni
+  PACKAGE_GUID                   = 0E9D6868-7D81-4A21-9785-BA1021F3F63B
+  PACKAGE_VERSION                = 0.10
+
+
+[Includes.common.Private]
+  Include/Private
```

Comment:
Do not put Private below Include.  When a package has both public and private interfaces, the public interfaces are typically below Include.  This means consumers of the package can still access the Private dirs with <Private/Protocol/Protocol.h>.  If the Private path is at the top level such as PrivateInclude, then callers can not get to the include files.

Reply:
Oh that's a great point, I'll fix this now